chart/templates/secrets/pgbouncer-certificates-secret.yaml

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+################################
+## Pgbouncer Certificate Secret
+#################################
+{{- if .Values.pgbouncer.enabled }}
```

Comment:
This would create an empty secret if the ssl file options aren't set - would be nice to avoid that

Reply:
True. Let me see how I can get rid of the secret entirely.

Reply:
Not an expert in go templating but  I think or on all certs/keys will work much better here. Seems that or on empty/null values works as expected.